src/actions.cr

```diff
@@ -40,4 +45,6 @@ module DppmRestApi::Actions
   def self.deny_access!(to context)
     raise Unauthorized.new context
   end
+
+  protected class_property? has_access : Proc(HTTP::Server::Context, Access, Bool) = ->(context : HTTP::Server::Context, access : Access) { false }
```

Comment:
I like the idea here -- allowing overriding the block to change how access is granted (for testing I assume)... but I think the implementation could use some improvement. Instead of calling `#call` on the proc directly, I think the property should be called `access_filter` or something to that effect, has_access? can be like
```crystal
def has_access?(context, access)
  access_filter.call context, access
end
```
Secondly, I think the property should be on the authentication middleware, so we should move `DppmRestApi::Actions.auth_handler` to `DppmRestApi::Authentication.handler`. Then there can be an `Authentication.has_access?` next to it.